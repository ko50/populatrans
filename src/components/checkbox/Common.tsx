import styles from "styles/components/checkbox/common.module.scss";

export const Checkbox: React.FC<
  React.InputHTMLAttributes<HTMLInputElement>
> = ({ ...props }) => {
  return <input className={styles.checkbox} type="checkbox" {...props} />;
};
