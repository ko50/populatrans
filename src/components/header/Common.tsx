import styles from "styles/components/header.module.scss";

export const Header: React.FC = () => {
  return <div className={styles.header}>POPULATrans</div>;
};
