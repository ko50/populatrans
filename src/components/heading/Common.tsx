type Props = {
  text: string;
};

export const H1: React.FC<Props> = ({ text }) => {
  return (
    <div>
      <h1>{text}</h1>
      <span />
    </div>
  );
};
