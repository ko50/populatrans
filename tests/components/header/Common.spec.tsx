import React from "react";
import renderer from "react-test-renderer";
import { Header } from "components/header/Common";

describe("common header component", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
