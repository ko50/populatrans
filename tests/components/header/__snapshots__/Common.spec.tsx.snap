// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common header component should render correctly 1`] = `
<div
  className="header"
>
  POPULATrans
</div>
`;
